### ISSUE TITLE
Setting Up Jasmine With ASP.Net Core Web Application

### ISSUE THREAD

**Issue by daryllabar** [jasmine/jasmine#1556]
Is there a recommended practice or a walk through for setting up Jasmine with an ASP.Net Core Web Application? [ I've posted a question on SO](https://stackoverflow.com/questions/50256118/setting-up-jasmine-with-asp-net-core-web-application), but haven't gotten any responses yet, so any type of response would be helpful.

**Comment by slackersoft** [comment]
In previous projects where we've used ASP.Net, we used [chutzpah](https://github.com/mmanela/chutzpah) to execute our specs. I don't remember all of the particulars of the setup, and there are a few quirks, but it hooks in and serves your javascript the same way that it will be in production, including the bundling. I'm not sure about Chutzpah's support for .Net Core though.

Next time, you are more likely to get quicker responses from the community for "How to use jasmine?" questions and a history of other solutions on the [jasmine-js group](https://groups.google.com/group/jasmine-js). We try to keep jasmine's github issues list focused on bugs and feature requests for jasmine itself.

Closing. Hope this helps. Thanks for using Jasmine!
